shared/versions/vaadin-core-versions.json

```diff
@@ -0,0 +1,385 @@
+{
+    "bundles": {
+        "vaadin": {
+            "jsVersion": "24.4.0-alpha13",
+            "npmName": "@vaadin/bundles"
+        }
+    },
+    "core": {
+        "a11y-base": {
+            "jsVersion": "24.4.0-alpha13",
+            "npmName": "@vaadin/a11y-base",
+            "mode": "lit"
+        },
+        "accordion": {
+            "javaVersion": "24.4-SNAPSHOT",
+            "jsVersion": "24.4.0-alpha13",
+            "npmName": "@vaadin/accordion",
+            "mode": "lit"
+        },
+        "app-layout": {
+            "javaVersion": "24.4-SNAPSHOT",
+            "jsVersion": "24.4.0-alpha13",
+            "npmName": "@vaadin/app-layout",
+            "mode": "lit"
+        },
+        "avatar": {
+            "javaVersion": "24.4-SNAPSHOT",
+            "jsVersion": "24.4.0-alpha13",
+            "npmName": "@vaadin/avatar",
+            "mode": "lit"
```

Comment:
package.json was updated before too based on `vaadin*-versions.json` which can be also excluded (core and pro components). But the difference is now that only having specific component is not enough after NpmPackage annotations are removed. This would have impact in projects that don't use `vaadin` or `vaadin-core` dependencies. They would need to add NpmPackage annotation somewhere or add it in package.json manually.

Reply:
Interesting and unexpected breaking change

Reply:
This might go like this for beta to fix vaadin/flow#18684, but breaking use case is noted and will get better solution. Removal of NpmPackage is very likely just temporary. Keeping it and even adding new properties may be required after all.